lib/api/core/models/plugin/data.js

```diff
@@ -0,0 +1,3 @@
+class Data {}
```

Comment:
hum.. I don't like terms like `utils` or `data`. Can we find a better way to handle it?

If we expose a constructor (cf my remark on plugin context), maybe we can let the plugin creator inject the constructor of the objects they want to manipulate in their collections?